### ISSUE TITLE
Multiple but different `brctl` utilities (missing MacOS version)

### ISSUE THREAD

**Issue by danemacmillan** [sigoden/argc-completions#53]
There are two known `brctl` utilities.

The first is linux-based and definitely the more well-known of the two. It is the "ethernet bridge administration" utility that is already part of this repo:

- https://github.com/sigoden/argc-completions/blob/main/completions/brctl.sh
- https://wiki.linuxfoundation.org/networking/bridge

Then there is the iCloud Drive utility, which is less well-known but available to every MacOS installation by default, though it is not part of this repo. It gives access to some low-level actions to manipulate iCloud Drive from the command line, such as "evicting" files and directories or downloading them again if they have been "optimized" (offloaded from hard drive).

I'm not quite sure how the MacOS utility could be included and disambiguated, or if even there is a desire to do it. Nevertheless, there are numerous obscure MacOS utilities that are already included, like `hdiutil`, `launchctl`, `caffeinate`, and `defaults`, so it would make sense for an important one like `brctl` to be included as well, since iCloud Drive is ubiquitous to the MacOS experience. 

The official man page:

```
BRCTL(1)                    General Commands Manual                   BRCTL(1)

NAME
     brctl – Manage the CloudDocs daemon

SYNOPSIS
     brctl ⟨command⟩ [command-options and arguments]

DESCRIPTION
     brctl understands the following commands:

     diagnose [options] [⟨diagnosis-output-path⟩]
         diagnose and collect logs

         -M,--collect-mobile-documents[=<container>]  (default: all
     containers)
         -s,--sysdiagnose     Do not collect what's already part of
     sysdiagnose
         -n,--name=<name>     Change the device name
         [<diagnosis-output-path>]
                              Specifies the output path of the diagnosis; -n
     becomes useless.

     download ⟨path⟩
         download a local copy of the document at this path

     evict ⟨path⟩
         evict the local copy of the document at this path

     log [options] [⟨command⟩]

         -c,--color[={yes,no}]
                              turn on or off color use
         -d,--path=<logs-dir> use <logs-dir> instead of default
         -H,--home=<home-dir> use this as the ~ prefix, to look for ~/L/
         -f,--filter=<predicate>
                              only show lines matching predicate
         -m,--multiline[={yes,no}]
                              turn on or off multiple line logging
         -n=<number>          number of initial lines to display
         -p,--page            use paging
         -w,--wait            wait for new logs continuously (syslog -w)
         -t,--shorten         Shorten UUIDs, paths, etc
         -s,--digest          Only print digest logs

     dump [options] [⟨container⟩]
         dump the CloudDocs database

         -o,--output=<file-path>
                              redirect output to <file-path>
         -d,--database-path=<db-path>
                              Use the database at <db-path>
         [<container>]        the container to be dumped

     monitor [options] ⟨container⟩
         use NSMetadataQuery to monitor the container

         -S,--scope=<scope>
                              restrict the NSMDQ scope to DOCS, DATA, or BOTH

     versions [options] ⟨path⟩ [ALL|etags...]
         list the non-local versions of the document at this path.

         -a,--all             List all non-local versions including those that
                              are locally cached

SEE ALSO
     bird(8)

Mac OS X                           22/04/14                           Mac OS X
```

**Comment by sigoden** [comment]
solved in #55

**Comment by danemacmillan** [comment]
Nice--thanks!
